installer/roles/local_docker/tasks/upgrade_postgres.yml

```diff
@@ -1,26 +1,31 @@
 ---
-- name: Check for existing Postgres data
-  stat:
-    path: "{{ postgres_data_dir }}/pgdata/PG_VERSION"
+
+- name: Register full path to postgres data dir
+  shell: "echo {{ postgres_data_dir }}"
+  register: fq_postgres_data_dir
+
+- name: Register temporary docker container
+  set_fact:
+    container_command: "docker run -v '{{ fq_postgres_data_dir.stdout }}:/var/lib/postgresql' centos:8 bash "
+
+- name: Check for existing Postgres data (run from inside the container for access to file)
+  shell:
+    cmd: "{{ container_command }} [[ -f /var/lib/postgresql/10/data/PG_VERSION ]] && echo 'exists'"
```

Comment:
I think that using https://docs.ansible.com/ansible/devel/collections/ansible/builtin/stat_module.html would be a more idiomatic Ansible thing to do here.

Reply:
@jbradberry is there a good way to stat this file from inside the container with ansible?  The issue is that we can't run this from the host without sudo otherwise.

Reply:
ansible has a docker connection module https://docs.ansible.com/ansible/latest/scenario_guides/guide_docker.html#connecting-to-the-docker-api

Reply:
I attempted to do this here: https://github.com/rooftopcellist/awx/compare/pg12-upgrade-migrations...rooftopcellist:pg12-upgrade-migrations-docker?expand=1

But ran into 2 issues:
1. If one of these tasks fail, the installer will leave behind a tmp container which will cause errors in later runs.  (Not that we couldn't solve this one, it would just make the playbook messy.  
2. I am running in to permission (?) issues when using this method.  
```
TASK [local_docker : Check for existing Postgres data (run from inside the container for access to file)] ************************************************************************************************************************************
task path: /home/chadams/awx/installer/roles/local_docker/tasks/upgrade_postgres.yml:13
fatal: [localhost]: UNREACHABLE! => {"changed": false, "msg": "Failed to create temporary directory.In some cases, you may have been able to authenticate and did not have permissions on the target directory. Consider changing the remote tmp path in ansible.cfg to a path rooted in \"/tmp\", for more error information use -vvv. Failed command was: ( umask 77 && mkdir -p \"` echo ~/.ansible/tmp `\"&& mkdir \"` echo ~/.ansible/tmp/ansible-tmp-1610982431.4560068-986689-254466671929443 `\" && echo ansible-tmp-1610982431.4560068-986689-254466671929443=\"` echo ~/.ansible/tmp/ansible-tmp-1610982431.4560068-986689-254466671929443 `\" ), exited with result 1", "unreachable": true}
```

I think we should merge what is currently in the PR.